### DISCUSSION TITLE
Installing Oqtane on IIS

### DISCUSSION THREAD

**Discussion by cherylbearden** [oqtane/oqtane.framework#1296] [Q&A] [answered]
I can follow the blog and download and install the install version of Oqtane on IIS and this works ok: 
https://www.oqtane.org/Resources/Blog/PostId/542/installing-oqtane-on-iis

But is it also possible to deploy the version of Oqtane that I have been working with in development to IIS? I see that the Publish option in Visual Studio does not work. 

So in order to deploy Oqtane on IIS do I always need to start with the latest Install package and then add my individual themes and modules to this?  Thanks very much.

**Comment by trapias** [comment]
Hello,
also about IIS, I am unable to run from a virtual application (like root site/oqtane): should this work?
Thanks for any help,
Alberto.

**Comment by JustCallMeAD** [comment, marked as answer]
Not sure if you have read Shaun's blog below but if you haven’t it may be well worth the read:
https://www.oqtane.org/Resources/Blog/PostId/542/installing-oqtane-on-iis

**Reply by cherylbearden** [reply]
Yes, I have read this link and gotten it to work ok.  But my question is, how to I make an installation package that works on IIS using the code in my development environment?  Thanks. :)

**Reply by leigh-pointer** [reply]
@cherylbearden What you will need to do is change your debug.cmd and release.cmd file in the packages project. the change you will need to do that esure that the files are built in the Root of the IIS application folder.  Then you are ready to go.
Make sure to build you module for release and run Oqtane first. Then you have your module registered.

**Reply by cherylbearden** [reply]
Thanks, I was able to get my development version hosted on IIS by overlaying the oqtane.* files in the Root of the IIS application folder.  

But all I really needed to do was get Telerik working in my Oqtane installation.  So I found information on how to do this [here](https://github.com/oqtane/oqtane.framework/discussions/690).

Also, for my theme I was able to install the Oqtane Bootswatch Theme and easily modify the colors in one of the existing themes. 

So I was able to get what I needed working while keeping the original Oqtane Framework Install solution.

**Reply by leigh-pointer** [reply]
@cherylbearden good to hear :)

**Comment by iJungleboy** [comment]
BTW: I created a video walkthrough for the whole process

https://2sxc.org/en/blog/post/install-oqtane-2-from-scratch-with-language-packs-and-2sxc-video

**Reply by cherylbearden** [reply]
This was an excellent video walkthrough.  Thanks!

**Reply by iJungleboy** [reply]
Glad you like it - please share around (it took more than 2 days of work to prepare, so the more it's watched, the more it's worth it)

**Comment by cherylbearden** [comment]
I will share it in our IT office at UF Research today!

________________________________________
